velvetarchangel/Library_Full_Stack_App
# Connecting frontend with backend

Connecting front end to backend.

Testing instructions:
1. go to /server -> npm run start
2. go to /client -> npm run start
3. go to localhost:3000 (frontend) and scroll to bottom, should be able to see text "api is working correctly"
4. go to localhost:5001/testAPI and see a response
